app/src/main/java/io/github/boogiemonster1o1/eyeyoureadyforit/data/EyeEntry.java

```diff
@@ -1,38 +1,24 @@
 package io.github.boogiemonster1o1.eyeyoureadyforit.data;
 
-import com.fasterxml.jackson.annotation.JsonCreator;
-import com.fasterxml.jackson.annotation.JsonProperty;
+import io.github.boogiemonster1o1.eyeyoureadyforit.App;
+import io.github.boogiemonster1o1.eyeyoureadyforit.util.DataSource;
 
-import java.nio.file.Path;
-import java.sql.DriverManager;
-import java.sql.Connection;
-import java.sql.Statement;
-import java.sql.ResultSet;
-import java.sql.SQLException;
 import java.util.ArrayList;
-import java.util.Arrays;
 import java.util.List;
 import java.util.Random;
 import java.util.concurrent.ThreadLocalRandom;
-import java.util.stream.Collectors;
 
 public final class EyeEntry {
 	private static final Random RANDOM = new Random(ThreadLocalRandom.current().nextLong());
-	private static final List<EyeEntry> ENTRIES = new ArrayList<>();
+	private static List<EyeEntry> ENTRIES = new ArrayList<>();
 	private final String imageUrl;
 	private final String name;
 	private final String hint;
 	private final List<String> aliases;
 
-	@JsonCreator
```

Comment:
how old is this
concern
the current version doesn't have JsonCreator jank does it?

Reply:
wtf i thought i removed that

Reply:
ok i did for sure, check the latest commit

Reply:
kek ok